src/pages/List.js

```diff
@@ -7,13 +7,13 @@ export default function List(props) {
   });
   return (
     <>
-      <h1>THIS IS THE LIST</h1>
-      <h2>Your token: {props.token}</h2>
+      <h1>This Is Your Grocery List</h1>
+      <h2>It uses the token: {props.token}</h2>
       {error && <strong>Error: {JSON.stringify(error)}</strong>}
-      {loading && <span>Collection: Loading...</span>}
+      {loading && <span>Grocery List: Loading...</span>}
       {listItem && (
         <>
-          <span>Collection:</span>
+          <span>Grocery List:</span>
           <ul>
             {listItem.docs.map((doc) => (
               <li key={doc.id}>{JSON.stringify(doc.data())}</li>
```

Comment:
Your alert on Home.js is only triggered when a user creates a new token.
The AC states:

> **The list view, when there are no items to display,** should show a prompt...

When the `listItem` is empty, there should be some prompt here to instruct the user how to add items to their list.
![1646aa24-a354-447c-9ae0-5013718fd72d](https://user-images.githubusercontent.com/6759658/116794843-1a16b480-aa85-11eb-9ca2-1ee9372c8851.gif)

Reply:
Got it! 

Prompt now exists on list view for user to add first item if list is empty.